docs/src/_includes/components/_globals.html

```diff
@@ -0,0 +1,63 @@
+{# to npm #}
+{% set toNpmCommands = {
+   "install": "install",
+   "test": "test",
+   "run": "run",
+   "init": "init",
+   "create": "create"
```

Comment:
```suggestion
{% set toNpmCommands = {
   "install": "install",
   "test": "test",
   "run": "run",
   "init": "init",
   "init-create": "init"
```
![Screenshot 2024-09-24 171523](https://github.com/user-attachments/assets/c6937d9a-b826-41f9-a60d-127e90a41e6d)

here npm has `init` and others have `create` command and also we haven't used `create` key anywhere in the docs right now